document/core/exec/instructions.rst

```diff
@@ -1126,21 +1126,20 @@ All these notions are made precise in the :ref:`Appendix <soundness>`.
 Expressions
 ~~~~~~~~~~~
 
-An :ref:`expression <syntax-expr>` is *evaluated* relative to a :ref:`current <exec-notation-textual>` :ref:`frame <syntax-frame>` pointing to its containing :ref:`module instance <syntax-moduleinst>`.
+An :ref:`expression <syntax-expr>` is executed relative to a :ref:`current <exec-notation-textual>` :ref:`frame <syntax-frame>` pointing to its containing :ref:`module instance <syntax-moduleinst>`.
 
 1. Jump to the start of the instruction sequence :math:`\instr^\ast` of the expression.
 
-2. Execute of the instruction sequence.
```

Comment:
I think you need to leave in the other steps, otherwise its not defined at the use sites what the "result" of evaluation is. Likewise, reinstantiate the `*evaluated*` above, because that's the name of the definition used elsewhere.

Reply:
Okay, I see what you have in mind.